examples/Encrypted Inference-LinearRegression.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
<p>TYPO: is the process</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/SyMPC/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Resolved.